### ISSUE TITLE
HsOpenSSL 0.11.5 compiling issues with GHC 8.2.2

### ISSUE THREAD

**Issue by flobrosch** [vshabanov/HsOpenSSL#53]
Hi, the latest release won't compile with all ghc versions:

https://matrix.hackage.haskell.org/#/package/HsOpenSSL

```
OpenSSL/Session.hsc:474:52: error:
    • Variable not in scope: (<>) :: IO a0 -> [Char] -> t0
    • Perhaps you meant one of these:
        ‘<$>’ (imported from Prelude), ‘*>’ (imported from Prelude),
        ‘<$’ (imported from Prelude)
    |
474 |                      throwIO $ ProtocolError $ loc <> ": "
    |                                                    ^^
OpenSSL/Session.hsc:475:24: error:
    • Variable not in scope: (<>) :: t0 -> String -> IO a
    • Perhaps you meant one of these:
        ‘<$>’ (imported from Prelude), ‘*>’ (imported from Prelude),
        ‘<$’ (imported from Prelude)
    |
475 |                        <> sslErrorString sslErr
    |                        ^^
OpenSSL/Session.hsc:499:29: error:
    • Variable not in scope: (<>) :: [Char] -> String -> String
    • Perhaps you meant one of these:
        ‘<$>’ (imported from Prelude), ‘*>’ (imported from Prelude),
        ‘<$’ (imported from Prelude)
    |
499 |   _ -> "Unknown SSL error " <> show e
    |                             ^^
```

**Comment by hvr** [comment]
In my function as Hackage Trustee, I've corrected the metadata for this release accordingly as this was starting to break every reverse dependency of HsOpenSSL on Hackage for GHC 8.2.2 and older:

https://hackage.haskell.org/package/HsOpenSSL-0.11.5/revisions/

before revision: https://matrix.hackage.haskell.org/package/HsOpenSSL@1604396398

after revision: https://matrix.hackage.haskell.org/package/HsOpenSSL@1604480957

This should be fairly easy to fix in HsOpenSSL to restore support for base < 4.11, which didn't yet export `<>` from `Prelude` by explicitly importing it.

**Comment by vshabanov** [comment]
Thank you for reporting this. Fixed in `HsOpenSSL-0.11.5.1` (just switched to `++`).
